front/app/components/PostShowComponents/Comments/ParentCommentForm.tsx

```diff
@@ -131,7 +146,9 @@ const ParentCommentForm = ({ postId, postType, className }: Props) => {
   const [hasApiError, setHasApiError] = useState(false);
   const [profanityApiError, setProfanityApiError] = useState(false);
   const [hasEmptyError, setHasEmptyError] = useState(true);
-
+  const [postAnonymously, setPostAnonymously] = useState(false);
+  const [showAnonymousConfirmationModal, setShowAnonymousConfirmationModal] =
+    useState(false);
```

Comment:
The "ParentCommentForm" and "ChildCommentForm" are very similar, and could probably be combined into 1 component at some point :thinking: But for now I've left them separate (which means a bit of code duplication).

Reply:
Yes, I had the same thought. Not sure what the rationale was for them being separate in the first place, but yes leave it for now.